affordable-housing/CSVToHTMLConverter.py

```diff
@@ -0,0 +1,148 @@
+import sys
+import csv
+import re
+import usaddress
+
+# @author: ashwin
+
+def stTypeToAbbrev(streetType):
+    if streetType == 'Avenue':
+        return 'AV'
+    elif streetType == 'Boulevard':
+        return 'BL'
+    elif streetType == 'Circle':
+        return 'CL'
+    elif streetType == 'Court':
+        return 'CT'
+    elif streetType == 'Drive':
+        return 'DR'
+    elif streetType == 'Expressway':
+        return 'EX'
+    elif streetType == 'Freeway':
+        return 'FY'
+    elif streetType == 'Highway':
+        return 'HY'
+    elif streetType == 'Lane':
+        return 'LN'
+    elif streetType == 'Loop':
+        return 'LP'
+    elif streetType == 'Place':
+        return 'PL'
+    elif streetType == 'Parkway':
+        return 'PY'
+    elif streetType == 'Road':
+        return 'RD'
+    elif streetType == 'Square':
+        return 'SQ'
+    elif streetType == 'Street':
+        return 'ST'
+    elif streetType == 'Terrace':
+        return 'TR'
+    elif streetType == 'Walkway':
+        return 'WW'
+    elif streetType == 'Way':
+        return 'WY'
+    else:
+        return streetType
+    
+
+def mapsUrlToHref(url, addressString):
+    retval = "<a href=\"" + url + " San Jose, CA\">" + addressString + "</a>"
```

Comment:
", San Jose, CA, \">", right? Addresses look like this:

123 Jones Rd, Cityville, AA